components/webui/imports/api/search/server/publications.js

```diff
@@ -66,7 +68,9 @@ Meteor.publish(Meteor.settings.public.SearchResultsCollectionName, ({
         ],
         limit: SEARCH_MAX_NUM_RESULTS,
         disableOplog: true,
-        pollingIntervalMs: COLLECTION_POLL_INTERVAL_MILLIS,
+        pollingIntervalMs: isActivelyPolling
+            ? COLLECTION_POLL_INTERVAL_MILLIS
+            : IDLE_INTERVAL_MILLIS,
```

Comment:
Can we leave this unconfigured if `false === isActivelyPolling`? i.e. `undefined` 

Our linter does not permit usage of `undefined` though. `null` might work but we have to test it out.

Alternatively, we can set it to an infinitely high number.